scene/gui/tree.cpp

```diff
@@ -5494,22 +5494,30 @@ void Tree::_do_incr_search(const String &p_add) {
 	ensure_cursor_is_visible();
 }
 
-TreeItem *Tree::_find_item_at_pos(TreeItem *p_item, const Point2 &p_pos, int &r_column, int &h, int &section) const {
+TreeItem *Tree::_find_item_at_pos(TreeItem *p_item, const Point2 &p_pos, int &r_column, int &r_height, int &r_section) const {
+	r_column = -1;
+	r_height = 0;
+	r_section = -1;
+
+	if (!root || !Rect2(Vector2(), get_size()).has_point(p_pos)) {
+		return nullptr;
+	}
```

Comment:
`_find_item_at_pos` shouldn't have the local rect check, AFAICT it takes in a position within the "unscrolled items area" (other methods before calling it account for the title and add scroll offsets, so it getting a position e.g. much bigger than size is valid/expected).

```suggestion
	if (!root) {
		return nullptr;
	}
```

Reply:
+It's an internal method so seems like `root` check is potentially not needed too. :thinking:

Reply:
Also reverted `_find_button_at_pos()` (the two methods are now consistent).